webpack/assets/javascripts/react_app/components/common/forms/Actions.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Button } from 'patternfly-react';
+
+import { noop } from '../../../common/helpers';
 import { simpleLoader } from '../Loader';
 import { translate as __ } from '../../../../react_app/common/I18n';
 
-export default ({ onCancel, disabled = false, submitting = false }) => (
+const FormActions = ({ onCancel, disabled = false, submitting = false }) => (
```

Comment:
are these assignments needed here? we have default values via proptypes.

Reply:
Nice catch, forgot to remove them :+1:

Reply:
Fixed